Makefile

```diff
@@ -78,6 +78,8 @@ fmt: herumi
 		--enable=misspell \
 		--enable=gosec
 
+replace_mutux:
```

Comment:
I think we don't need a new target here. Let's don't change the makefile. Better add a new workflow for only checking data race when user send a pull request. The report is enough for finding the data race condition.